modules/ossm-integrating-with-user-workload-monitoring.adoc

```diff
@@ -0,0 +1,165 @@
+////
+Module included in the following assemblies:
+* service_mesh/v2x/ossm-observability.adoc
+////
+
+:_content-type: PROCEDURE
+[id="ossm-integrating-with-user-workload-monitoring_{context}"]
+= Integrating with user-workload monitoring
+
+By default, {SMProductName} (OSSM) installs the Service Mesh Control Plane (SMCP) with a dedicated instance of Prometheus for collecting metrics from a mesh. However, production systems need more advanced monitoring systems, like {product-title} monitoring for user-defined projects.
+
+The following steps show how to integrate Service Mesh with User-Workload Monitoring.
+
+.Procedure
+
+. Enable monitoring in user-defined projects by adding the following configuration to the `cluster-monitoring-config` config map:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    enableUserWorkload: true
+----
+
+. Configure the Service Mesh control plane for User-Workload Monitoring:
++
+[source,yaml]
+----
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: basic
+  namespace: istio-system
+spec:
+  addons:
+    prometheus:
+      enabled:  false # <1>
+  meshConfig:
+    extensionProviders:
+    - name: prometheus # <2>
+      prometheus: {}
+  security:
+    manageNetworkPolicy: false # <3>
```

Comment:
> I believe our "default" for the documentation should be the custom NetworkPolicy method

Yes, I agree.

> The tl;dr reason for this is that turning off manageNetworkPolicy has broad implications

I wouldn't say so... Service Mesh users should favor Istio AuthorizationPolicy over kubernetes NetworkPolicy. But anyway, I agree that custom network policy is better than turning off mangeNetworkPolicy to avoid user concerns.